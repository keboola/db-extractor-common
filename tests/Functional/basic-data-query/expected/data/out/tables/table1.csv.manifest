{
  "destination": "table1",
  "incremental": false
}
