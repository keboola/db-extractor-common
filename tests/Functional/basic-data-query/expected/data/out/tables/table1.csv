"col1","col2"
"row1","r1"
"row2","r2"
